leafmap/heremap.py

```diff
@@ -14,6 +14,9 @@
 from .basemaps import xyz_to_heremap
 from .common import shp_to_geojson, gdf_to_geojson, vector_to_geojson, random_string
 from . import examples
+from typing import Optional, Union, List, Dict, Callable, Any, Tuple
```

Comment:
The heremap package has not update since October 2021. I am going to remove this module. 
https://github.com/heremaps/here-map-widget-for-jupyter